app/models/comment.rb

```diff
@@ -1,7 +1,6 @@
 class Comment < ApplicationRecord
   belongs_to :author, class_name: 'User'
   belongs_to :post
-
```

Comment:
- [ ]  It would be nice if you use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_save` to call the `#comments_count` method every time a new comment has been saved to database, then make the `comments_count` method private.

```ruby
 after_save :comments_count

  private

  def comments_count
    post.increment!(:comments_count)
  end
```